solution/ui/regulations/eregs-vite/package.json

```diff
@@ -14,13 +14,11 @@
     "lodash": "^4.17.21",
     "vue": "^3.5.11",
     "vue-router": "^4.4.5",
-    "vue-template-compiler": "^2.7.16",
     "vuetify": "^3.7.1"
   },
   "devDependencies": {
     "@vitejs/plugin-vue": "^5.1.4",
     "eslint-plugin-vuetify": "^2.4.0",
-    "sass": "^1.79.4",
```

Comment:
Since we're moving away from CSS-in-JS, we don't need to include `sass` as a dependency here anymore